src/shogun/mathematics/Math.h

```diff
@@ -1141,11 +1159,18 @@ class CMath : public CSGObject
 		/** Performs a bubblesort on a given matrix a.
 		 * it is sorted in ascending order from top to bottom
 		 * and left to right
-		 * @param a
-		 * @param cols
-		 * @param sort_col
+		 * @param a matrix to be sorted
+		 * @param cols number of columns
+		 * @param sort_col 
```

Comment:
I'm not sure what this parameter is doing exactly. Some help :) I don't seem to understand how the function is sorting. I know what is bubble sort. Not sure how the numbers are being swapped (vertically, horizontally or diagonally?)